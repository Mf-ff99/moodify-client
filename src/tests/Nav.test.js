import React from 'react'
import Nav from '../components/nav'

import { shallow } from 'enzyme'

it('renders without crashing', () => {
    shallow(<Nav />)
})