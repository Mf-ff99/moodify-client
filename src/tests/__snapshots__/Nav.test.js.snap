// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component  renders form approriately 1`] = `
<Fragment>
  <nav
    className="Header"
  >
    <ul>
      <li
        className="right"
      >
        <Link
          to="/"
        >
          Moodify
        </Link>
      </li>
      <li
        className="left"
      >
        <Link
          to="/login"
        >
          Log in
        </Link>
      </li>
      <li
        className="right"
      >
        <Link
          to="/register"
        >
          Register
        </Link>
      </li>
    </ul>
  </nav>
</Fragment>
`;
